source/contracts/reporting/Market.sol

```diff
@@ -51,6 +51,7 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
     IMailbox private marketCreatorMailbox;
     uint256 private finalizationTime;
     uint256 private noShowBond;
+    bool private disputePacingOn;
```

Comment:
disputePacingOn is a flag which dictates if a dispute round should wait for a new fee window or allow multiple disputes to take place within the same fee window. the idea here is that while early rounds can go through several rounds of dispute very quickly, it is best to give a bit of time to before getting closer to a fork